data/_ui/interfaces.txt

```diff
@@ -525,6 +529,55 @@ interface "plugins"
 
 
 
+interface "install plugins"
+	sprite "ui/plugins panel"
+		center -20 -20
+	button c "_Controls"
+		center -300 -230
+		dimensions 90 30
+		color medium
+		hover bright
+	button s "_Settings"
+		center -300 -190
+		dimensions 90 30
+		color medium
+		hover bright
+	button p "_Plugins"
+		center -300 -150
+		dimensions 90 30
+		color bright
+
+	active if "can install/update"
+	button i "_Install/Update"
+		center -195 210
+		dimensions 120 30
+	active if "installed plugin"
+	button d "_Delete"
```

Comment:
The Delete button should go on the installed plugin page too, not on the online plugin list page.

Reply:
yup